components/button/button-subtle.js

```diff
@@ -204,7 +211,7 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 				id="${this._buttonId}"
 				name="${ifDefined(this.name)}"
 				type="${this._getType()}">
-				${icon}
+				<slot name="icon" @slotchange="${this._handleIconSlotChange}"><d2l-icon icon="${ifDefined(this.icon)}" class="d2l-button-subtle-icon"></d2l-icon></slot>
```

Comment:
I still like conditionally rendering the `<d2l-icon>` vs. `nothing` if they don't provide an `icon` property. Technically I think subtle buttons could exist in the wild with no icon specified, and I'd hate to render a `<d2l-icon>` that did nothing in those cases.

Reply:
Agreed! I'll update this to be done the same as button-icon